crates/host/src/oci.rs

```diff
@@ -177,17 +186,15 @@ pub async fn fetch_actor(
 /// Returns an error if either fetching fails or reading the fetched OCI path fails
 #[allow(clippy::implicit_hasher)]
 pub async fn fetch_provider(
```

Comment:
Nit: `fetch_actor` uses `(creds, ref, opts)` for its arg order, but now `fetch_provider` uses `(ref, link, creds, opts)`. Can we be consistent?

Reply:
Unfortunate, I thought I made that change. Yes, I wanted it to be component information -> creds -> opts

Reply:
Implemented in https://github.com/wasmCloud/wasmCloud/pull/496